docs/components/oculus-touch-controls.md

```diff
@@ -34,34 +35,43 @@ mappings, events, and a Touch controller model.
 | triggerup            | Trigger released.              |
 | triggertouchstart    | Trigger touched.               |
 | triggertouchend      | Trigger no longer touched.     |
+| triggerchanged       | Trigger changed.               |
```

Comment:
good to know, thanks.  this code needs to support other browsers that currently lack those, let's revisit when more widely available?